src/index.html

```diff
@@ -3,12 +3,74 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+        class="logo_img"
+        src="images/logo.png"
+        alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav-item">
+            <a class="nav_links is-active"
+            href="index.html">Apple
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav_links"
+            href="index.html">Samsung
+            </a>
+          </li>
+          <li class="nav-item">
+            <a class="nav_links"
+            href="index.html">Smartphones
+            </a>
+          </li>
+          <li class="nav-item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav-item">
```